twins/network.go

```diff
@@ -130,6 +130,13 @@ func (n *Network) createTwinsNodes(nodes []NodeID, scenario Scenario, consensusN
 			return err
 		}
 
+		if pipelinedViews < 1 || consensusName != "chainedhotstuff" {
```

Comment:
This should not be done in the loop, and it is also more complex than necessary. Adding this at line 124 should suffice:
```go
	if pipelinedViews > 1 && consensusName != "chainedhotstuff" {
		return fmt.Errorf("pipelining currently only supported for chainedhotstuff")
	}
	if pipelinedViews < 1 {
		pipelinedViews = 1
	}
```

Reply:
fixed in [fb04815](https://github.com/relab/hotstuff/pull/78/commits/fb04815f86cc71243e680d698634e0b055489b3b)
I also added a similar error in `internal/cli/run.go`
```go
func checkFlags() error {
	consensus := viper.GetString("consensus")
	pipelinedViews := viper.GetInt("pipelined-views")
	if pipelinedViews > 1 && consensus != "chainedhotstuff" {
		return errors.New("piplining only supported for chainedhotstuff consensus")
	}
	return nil
}
```